src/query/api/v1/middleware/rewrite.go

```diff
@@ -88,7 +90,7 @@ func rewriteRangeDuration(
 	logger *zap.Logger,
 ) error {
 	// Extract relevant query params
-	query, start, end, err := extractParams(r, opts.Instant)
+	query, start, end, lookback, isLookackSet, err := extractParams(r, opts.Instant)
```

Comment:
nit: typo in `isLookackSet`

Reply:
https://github.com/m3db/m3/pull/3882/commits/bcf48423c7fa0f20d0f80c8f4ea44532c5c4d8e8#diff-f284760ffa1aa45c70f82bdf3d8982cc7328962c64802875e8cf7c5dd9c61784R150-R151